hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/api/TokenServiceApiImpl.java

```diff
@@ -282,46 +280,60 @@ public void updateStorageMetadata(
                 .build());
     }
 
-    private void maybeRemoveAlias(@NonNull final WritableAccountStore store, @NonNull final Bytes alias) {
-        if (!Bytes.EMPTY.equals(alias)) {
-            store.removeAlias(alias);
-        }
+    @Override
+    public void chargeNetworkFee(@NonNull AccountID payerId, long amount, @NonNull FeeRecordBuilder rb) {
+        requireNonNull(rb);
+        requireNonNull(payerId);
+
+        final var payerAccount = lookupAccount("Payer", payerId);
+        final var amountToCharge = Math.min(amount, payerAccount.tinybarBalance());
+        chargePayer(payerAccount, amountToCharge);
+        rb.transactionFee(rb.transactionFee() + amountToCharge);
```

Comment:
Agreed this is confusing, and only works because `rb.transactionFee()` should always be `0` going in to this method.

Removed the `rb.transactionFee() +` term.